python/tvm/relay/prelude.py

```diff
@@ -16,8 +16,516 @@
 # under the License.
 # pylint: disable=no-else-return, unidiomatic-typecheck, invalid-name
 """A prelude containing useful global functions and ADT definitions."""
+from .ty import GlobalTypeVar, TensorType, Any, scalar_type
+from .expr import Var, Function, GlobalVar, If, const
+from .op.tensor import add, subtract, equal
+from .adt import Constructor, TypeData, Clause, Match
+from .adt import PatternConstructor, PatternVar, PatternWildcard
+from . import op
 from .module import Module
 
+class TensorArrayOps:
+    """Contains tensor array related ops"""
+
+    def __init__(self, prelude, dtype):
+        """Create tensor array ops registry"""
+        self.prelude = prelude
+        self.dtype = dtype
+
+    def get_name(self, canonical):
+        """Get name corresponding to the caninical name"""
+        if canonical == 'tensor_t':
+            return 'tensor_{}_t'.format(self.dtype)
+        return "{}_{}".format(canonical, self.dtype)
+
+    def get_var(self, canonical):
```

Comment:
I feel it's better if we move `get_var` to Prelude so that we don't need to create TensorArrayOps each time we need to use it.